svelte-kit-scss/src/lib/components/Repo/RepoHeader/RepoNavigation/RepoNavigation.svelte

```diff
@@ -39,7 +43,8 @@
         on:click={openCode}
         on:keypress={openCode}
         class="tab tab--inactive"
-        class:tab--active={!path.includes('issues') && !path.includes('pull-requests')}
+        class:tab--active={!path.includes(issueSarchPageTypeIssue) &&
+          !path.includes(issueSarchPageTypePullRequest)}
```

Comment:
Wouldn't these be better as links?

Then we could get rid of all this JS right?